### PR TITLE
Verifying that script is being execute from main/scripts folder

### PR DESCRIPTION
**Pull request by vishipayyallore**
1. Verifying that script is being executed from the main/scripts folder

image

### CODE REVIEW

**Review by apaarshrm39 — COMMENTED** [review]
approved

**Comment by vishipayyallore** [comment]
> approved

Thanks @apaarshrm39

**Review by apaarshrm39 — APPROVED** [review]
:)

File: Scripts/git-clone-pull.sh [hunk 1]
```diff
@@ -4,14 +4,20 @@
 # Description: Shell script to clone a git repository and pull the latest changes
 # Authors: Robbie, and Swamy
 # Date: 12-Mar-2022
-# Modified: 14-Mar-2022
+# Modified: 15-Mar-2022
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # 
 
+if [[ "/$PWD/" != *"/eshop-main/Scripts"* ]]; then
```
Comment (robbiemu) [inline Scripts/git-clone-pull.sh:10]: if we are warning people to execute the script from Scripts, the trailing star is not needed

File: Scripts/git-clone-pull.sh [hunk 2]
```diff
@@ -4,14 +4,20 @@
...
+	echo -e "\e[1;31m Please execute the script from (YourDrive/YourRootFolder/eshop-main/Scripts) directory \e[0m"
+
+	exit
+fi
+
 while [[ "/$PWD/" = *"/eshop-main/"* ]]; do
```
Comment (robbiemu) [inline Scripts/git-clone-pull.sh:16]: we could remove this action if we advised them to start from the appropriate directory

**Pull request merged**